platform/ios/test/MGLAnnotationViewTests.swift

```diff
@@ -0,0 +1,23 @@
+import XCTest
+import Mapbox
+
+class CustomAnnotationView: MGLAnnotationView {
+    
+    override init(reuseIdentifier: String?) {
+        super.init(reuseIdentifier: reuseIdentifier)
+    }
+    
+    required init?(coder aDecoder: NSCoder) {
+        super.init(coder: aDecoder)
+    }
+    
+}
+
+class MGLAnnotationViewTests: XCTestCase {
```

Comment:
If I'm not mistaken, we already have a bridging header and some Objective-C-Swift combo test classes. But renaming is reasonable too.